pkg/action/executor/resize_container_util.go

```diff
@@ -168,81 +147,118 @@ func genMemoryQuantity(newValue float64) (resource.Quantity, error) {
 	if tmp < 1 {
 		tmp = 1
 	}
-	result, err := resource.ParseQuantity(fmt.Sprintf("%dKi", tmp))
-	if err != nil {
-		glog.Errorf("failed to generate Memory quantity: %v", err)
-		return result, err
+	return resource.ParseQuantity(fmt.Sprintf("%dKi", tmp))
+}
+
+func resizeContainer(client *kclient.Clientset, pod *k8sapi.Pod, spec *containerResizeSpec,
+	consistentResize bool) (*k8sapi.Pod, error) {
+	if consistentResize {
+		return nil, resizeControllerContainer(client, pod, spec)
 	}
+	return resizeSingleContainer(client, pod, spec)
+}
 
-	return result, nil
+// Consistent resize of container pods that belong to a controller
+func resizeControllerContainer(client *kclient.Clientset, pod *k8sapi.Pod, spec *containerResizeSpec) error {
+	// prepare controllerUpdater
+	controllerUpdater, err := newK8sControllerUpdater(client, pod)
+	if err != nil {
+		glog.Errorf("Failed to create controllerUpdater: %v", err)
+		return err
+	}
+	glog.V(2).Infof("Begin to resize %v for pod %s/%s.",
```

Comment:
I have updated the wording a little bit, instead of saying 
> Begin to resize Deployment for pod xxx, 

I changed it to 

>Begin to consistently resize Deployment of pod xxx" to make it more clear.